EventsExpress/ClientApp/src/components/event/event-filter-status.js

```diff
@@ -36,7 +51,10 @@ class EventFilterStatus extends Component {
                                 }
                                 return input.onChange(newValue);
                             }} />
-                        {eventStatusHistoryReadingString(option)}
+                        {options.length >= 4 
```

Comment:
Please, pass options from outside of this component, rather than doing unobvious logic inside.